.eslintrc.json

```diff
@@ -9,5 +9,6 @@
   },
   "extends": [
     "eslint:recommended"
-  ]
+  ],
+  "ignorePatterns": ["jest.config.js", "**/test/*.test.js", "module_shim.js"]
```

Comment:
What's the motivation for excluding test files here? I don't mind and I'm sure you have a reason, but I feel like I could learn something! :)

Reply:
I was getting no-undef errors jest globals (describe, it, etc.). 
Now thinking back, I could probably exclude them with `/* globals .... */ ` comment and do not ignore the test file :)

Reply:
Added them to "globals" in eslint config instead

Reply:
Ah, cool. :)